initial_setup.py

```diff
@@ -62,19 +66,34 @@ def get_answer(question, proper_answers=('yes','no')):
     return answer
 
 
-print "*" * TERMINAL_WIDTH
```

Comment:
I converted these python 2 style `print` statements into calls to the `print()` function in python 3.